Dockerfile

```diff
@@ -62,11 +62,16 @@ ENV WORKSPACE=$HOME/fabric8-planner
 RUN mkdir $WORKSPACE
 
 COPY . $WORKSPACE
-RUN chown -R ${FABRIC8_USER_NAME}:${FABRIC8_USER_NAME} $HOME/*
```

Comment:
Why the user is removed? Will the app be run inside container using root user?

Reply:
The user isn't removed, this is just forcing recursive file permissions (ownership) onto the created user.
This avoids too many corner cases (and cuts down build time by 5min).

Unless we have very good reason, I'm sure creating and switching to another user only adds complexity in the setup. Using `root` user works perfectly fine in this sandboxed environment.